README.md

```diff
@@ -1,2 +1,82 @@
 # queries
 Hosts reusable log queries which are built into a single queries.json file.
+
+
+# Query database structure
+
+```yaml
+queries:
+  query1:  # that is query id (sorted by id)
+    name: foo  # required
+    regex: bar  # optional
+    pattern: aaa # optional (either regex or pattern required)
+    url: ... # str or list[str], issue links to lp, bz,... (optional)
+    category: str  # required, one of ...
+    tags: []  # lists of tags (optional)
+    suppress-graph: false  # used for e-r (optional)
+    # uncommon fields
+    multiline: true  # https://opendev.org/openstack/ansible-role-collect-logs/src/branch/master/vars/sova-patterns.yml#L47
+   query2: ...
+```
+
+Both [elastic-search](https://www.elastic.co/guide/en/elasticsearch/reference/current/term-level-queries.html) and artcl can make use of `regex` searches.
+
+Pattern is supposed to be an exact string match and if multiple are present
+we could easily convert them into a regex that uses and.
+
+## Pattern
+
+On e-r queries we have multiple cases with multiple entries used on patterns,
```

Comment:
We know what e-r means, but in the documentation we should not abbreviate

Reply:
+1